pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -156,6 +156,21 @@ func (opt *Options) Run(suite *TestSuite, junitSuiteName string) error {
 		return err
 	}
 
+	discoveryClient, err := getDiscoveryClient()
```

Comment:
need to find a way for this to not fail when running a --dry-run without a kubeconfig.  I suggest allowing all in such a case.